webui/react/src/utils/task.ts

```diff
@@ -129,7 +129,7 @@ export const generateExperiments = (count = 30): Type.ExperimentItem[] => {
         resourcePool: `ResourcePool-${Math.floor(Math.random() * 3)}`,
         searcherType: 'single',
         username: user.username,
-      } as Type.ExperimentItem;
+      } as unknown as Type.ExperimentItem;
```

Comment:
Not a fan of this change. `generateExperiments` is for testing, so you ought to be able to add filler values if they are expecting `parentArchived: false` or other fields

Reply:
yeah, all the experiment-specific code in that file needs to be moved to `experiment.ts` and disentangled from the tasks.

this is just a placeholder for now.